server/src/service-docs/service-doc.dto.ts

```diff
@@ -0,0 +1,55 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CreateServiceDocRequest {
```

Comment:
@MBuchalik The API model for the `cli` and the `frontend`.
Any thoughts / wishes?

Reply:
I think this looks good for a first version 👍 

One thing I was not sure about: Is `deploymentDocumentation` and `apiDocumentation` expected to contain the actual documentation, or a URL to the documentation? (No need to change anything here, I guess we just need to document that somewhere.)

Reply:
For now an URL. But I like the idea of also uploading e.g. markdown. Maybe later, though. Also referencing a local file that the CLI includes into the upload might be interesting in that regard.

Reply:
It seems like the README needs to be adjusted, because in the example json document, a few fields are called differently.

| README | DTO |
| --- | --- |
| serviceName | name |
|consumedAPIs | usedAPIs|
| (missing) | providedAPIs|
|(missing) | repository|
|(missing) | taskBoard |
|technicalDocumentation|developmentDocumentation|
|(missing)|deploymentDocumentation|
|apiDocumentation)|(missing)|

Reply:
Good point! Will update the README in the afternoon.